spec/vc-11.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
+  "type": "object",
+  "definitions": {
+    "credentialStatus": {
+          "type": "object",
+          "properties": {
+            "id": { "type": "string" },
+            "type": { "type": "string" },
+            "statusPurpose": { "type": "string" },
+            "statusListIndex": { "type": "string" },
+            "statusListCredential": { "type": "string" }
+          },
+          "required": ["id", "type", "statusPurpose", "statusListIndex", "statusListCredential"]
+        },
+        "credentialSchema": {
+          "type": "object",
+          "properties": {
+            "id": { "type": "string" },
+            "type": { "type": "string" }
+          },
+          "required": ["id", "type"]
+        }
+  },
+  "properties": {
+    "@context": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "minItems": 1
+    },
+    "id": {
+      "type": "string"
+    },
+    "type": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "minItems": 1
+    },
+    "issuer": {
+      "oneOf": [
+        { "type": "string" },
```

Comment:
this will be our default when creating, but we will accept creds that are 
{ id: 123, name: myname} right?

currently if we get a jwt, we use what is in the iss for buildling the vc and adding the issuer

so if we get

iss: abc

vcDataModel.issuer: { id: abc, name: myname}

we will blast it and make vcDataModel.issuer: abc

Reply:
correct